spring-cloud-stream-reactive/src/main/java/org/springframework/cloud/stream/reactive/StreamEmitter.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2016-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.reactive;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.annotation.Input;
+import org.springframework.cloud.stream.annotation.Output;
+
+/**
+ * Method level annotation that marks a method to be an emitter to outputs declared via
+ * {@link EnableBinding} (e.g. channels).
+ *
+ * This annotation is intended to be used in a Spring Cloud Stream application that requires
+ * a source to write to one or more {@link Output}s using the reactive paradigm.
+ *
+ * No {@link Input}s are allowed on a method that is annotated with StreamEmitter.
+ *
+ * Depending on how the method is structured, there are some flexibility in how the {@link Output} may be used.
+ *
+ * Here are some supported usage patterns:
+ *
+ * A StreamEmitter method that has a return type cannot take any method parameters.
+ *
+ * &#064;StreamEmitter
```

Comment:
Hm.
All these must be wrapped to code to be readable from the target JavaDocs resource.
For example:
```
* <pre class="code">
 * &#064;Configuration
 * &#064;EnableIntegration
 * &#064;ComponentScan(basePackageClasses = { MyConfiguration.class })
 * public class MyIntegrationConfiguration {
 * }
 * </pre>
```
which results to the: http://docs.spring.io/spring-integration/api/org/springframework/integration/config/EnableIntegration.html